chpl/chpl-service/src/main/java/gov/healthit/chpl/FF4JConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package gov.healthit.chpl;
+
+import java.sql.SQLException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+import org.ff4j.FF4j;
+import org.ff4j.audit.repository.JdbcEventRepository;
+import org.ff4j.cache.FF4JCacheManager;
+import org.ff4j.cache.FeatureCacheProviderEhCache;
+import org.ff4j.property.store.JdbcPropertyStore;
+import org.ff4j.store.JdbcFeatureStore;
+import org.ff4j.store.JdbcQueryBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import gov.healthit.chpl.dao.impl.BaseDAOImpl;
+
+@Configuration
+public class FF4JConfiguration {
+    private static final String TABLE_PREFIX = "ff4j_";
+
+    @Bean
+    public FF4j getFF4j() {
+        try {
+            FF4j ff4j = new FF4j();
+
+            JdbcFeatureStore featureStore = new JdbcFeatureStore(ff4jDataSource());
+            featureStore.setQueryBuilder(new JdbcQueryBuilder(TABLE_PREFIX, "", BaseDAOImpl.SCHEMA_NAME));
```

Comment:
Does this add a ton of tables? Just wondering if we might rather have the tables in their own schema (like quartz is) rather than mixed in with all of our chpl/listing data.

Reply:
There are 5 tables that are created.  We could add them to a different schema.  There are several issues we need to figure out how we want to handle going forward.  I'll add this to the list of questions I have,

Reply:
Sounds good!